### ISSUE TITLE
Create a Proof of Concept Cypress Dashboard Parallelization Configuration

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#22830]
## User Story

Explore Cypress Dashboard as a E2E test parallelization solution.

## Expected Behavior
Cypress Dashboard is automagically breaking up our Cypress E2E test suite into optimized chunks of work and reduces the execution time of the E2E test suite to less than 20 minutes.

## Acceptance Criteria

- [ ] Cypress Dashboard configuration is successfully executing our Cypress E2E test suite in parallel

**Comment by pjhill** [comment]
Duplicative of a [separate ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22446) under which this effort was actually worked.
